Add missing import to docs upload script.

jdm opened the pull request.

jdm committed:
Add missing import to docs upload script.

SimonSapin commented:
@bors-servo r+

bors-servo commented:
:pushpin: Commit 8798c71 has been approved by `SimonSapin`

<!-- @bors-servo r=SimonSapin 8798c71f304384d2b373cda1b38f29e675149e14 -->

SimonSapin committed:
Split doc generation into its own task

This task reached its 60 minutes timeout in
https://community-tc.services.mozilla.com/tasks/YByC2V7uQq2lQv40TVi5KQ/runs/1/logs/live/https%3A%2F%2Fcommunity-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FYByC2V7uQq2lQv40TVi5KQ%2Fruns%2F1%2Fartifacts%2Fpublic%2Flogs%2Flive.log#L6276

bors-servo commented:
:hourglass: Testing commit 8798c71f304384d2b373cda1b38f29e675149e14 with merge 17fa176f03faf33b613376fc31314501a3e6be56...

bors-servo commented:
:broken_heart: Test failed - [status-community-tc](https://community-tc.services.mozilla.com/tasks/groups/eluz3CCHSRKwlMl8w3TRGQ)

SimonSapin commented:
https://github.com/servo/servo/pull/24683 (landing now) should fix this failure.

SimonSapin commented:
Rebase on top of that PR, to land them together

@bors-servo r+ p=2

bors-servo commented:
:pushpin: Commit 540b28e has been approved by `SimonSapin`

<!-- @bors-servo r=SimonSapin 540b28e44fbcbccaf645753819de95e6f03a91a1 -->

bors-servo commented:
:hourglass: Testing commit 540b28e44fbcbccaf645753819de95e6f03a91a1 with merge 933459860e472c791ca33ef0df619fb19788d8b3...

bors-servo commented:
:snowman: The build was interrupted to prioritize another pull request.

bors-servo commented:
:hourglass: Testing commit 540b28e44fbcbccaf645753819de95e6f03a91a1 with merge d9b26fb445f1d70d122893b542cd64a5ca516c3c...

bors-servo commented:
:broken_heart: Test failed - [status-taskcluster](https://tools.taskcluster.net/groups/bPy6tJQITL-sWYkDHM10Yg)

jdm commented:
@bors-servo retry

bors-servo commented:
:hourglass: Testing commit 540b28e44fbcbccaf645753819de95e6f03a91a1 with merge 5cedef45959be45f524493207a1caf71148a4fa5...

bors-servo commented:
:broken_heart: Test failed - [status-taskcluster](https://tools.taskcluster.net/groups/dj8UQozEQLewtwD9mfhVJA)

jdm commented:
@bors-servo retry

bors-servo commented:
:bomb: Failed to start rebuilding: `Unknown error`

bors-servo commented:
:hourglass: Testing commit 540b28e44fbcbccaf645753819de95e6f03a91a1 with merge fc7b88019538de63fdd4d24ed720ad7198602fd8...

bors-servo commented:
:sunny: Test successful - [linux-rel-css](http://build.servo.org/builders/linux-rel-css/builds/14068), [linux-rel-wpt](http://build.servo.org/builders/linux-rel-wpt/builds/14131), [status-community-tc](https://community-tc.services.mozilla.com/tasks/groups/D7YM9-TwSVuyoE-nIBN1Ow), [status-taskcluster](https://tools.taskcluster.net/groups/DQgPF9PqR6yc1bd2nL2Vag)
Approved by: SimonSapin
Pushing fc7b88019538de63fdd4d24ed720ad7198602fd8 to master...
